packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -3638,3 +3638,84 @@ it('should make column sticky and then non-sticky after dragging sticky column a
     expect(cell).not.toHaveStyle('--iui-table-sticky-left: 400px');
   });
 });
+
+it('should render start and end cell icons', () => {
+  const testColumns: Column<TestDataType>[] = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+          cellRenderer: (props) => {
+            return <DefaultCell {...props} startIcon={<SvgPlaceholder />} />;
+          },
+        },
+        {
+          id: 'description',
+          Header: 'description',
+          accessor: 'description',
+          cellRenderer: (props) => {
+            return <DefaultCell {...props} endIcon={<SvgPlaceholder />} />;
+          },
+        },
+      ],
+    },
+  ];
+  const { container } = renderComponent({
+    columns: testColumns,
+  });
+
+  const startIcon = container.querySelector('.iui-cell-start-icon');
```

Comment:
Would be nicer if you have more specific selectors here, so you actually check that first cell has only start icon and last cell has only end icon.
Also, we have some exapmles where we check the actual icon content to be sure, its same. So you could also pass different start and end icons to verify that.

Reply:
Updated tests to be more specific.